src/main/java/de/dafuqs/spectrum/mixin/compat/quilt_status_effect/absent/LivingEntityPreventStatusClearMixin.java

```diff
@@ -21,22 +23,42 @@ public abstract class LivingEntityPreventStatusClearMixin {
 	
 	@Shadow
 	public abstract void remove(Entity.RemovalReason reason);
+
+	@Shadow
+	public abstract boolean addStatusEffect(StatusEffectInstance effect);
+
+	@Shadow
+	public abstract Map<StatusEffect,StatusEffectInstance> getActiveStatusEffects();
 	
+	private boolean hasFatalSlumber = false;
+
+	@Inject(method = "clearStatusEffects", at = @At("HEAD"))
+	private void spectrum$detectFatalSlumber(CallbackInfoReturnable<Boolean> cir) {
+		hasFatalSlumber = getActiveStatusEffects().containsKey(SpectrumStatusEffects.FATAL_SLUMBER);
```

Comment:
This is injected into the same method; why not use MixinExtras' `@Share` instead?
FYI: [this](https://github.com/LlamaLad7/MixinExtras/wiki/Share)

Reply:
entirely on me, this is my first time doing something this complex and I had no idea that existed